src/test/java/org/takes/http/BkBasicTest.java

```diff
@@ -135,7 +134,71 @@ public void exec(final URI home) throws IOException {
     }
 
     /**
+     * BkBasic produces headers with addresses without slashes.
+     *
+     * @throws IOException If some problem inside
+     */
+    @Test
+    public void addressesInHeadersAddedWithoutSlashes() throws IOException {
+        final Socket socket = BkBasicTest.createMockSocket();
+        final AtomicReference<Request> ref = new AtomicReference<Request>();
```

Comment:
@alexey-krylov why do we need `AtomicReference` here?
`BkBasic` doesn't use threads as I know